README.md

```diff
@@ -1,10 +1,15 @@
 # noUiSlider
 
-noUiSlider is lightweight JavaScript range slider, originally developed to be a jQuery UI alternative. It features cross-browser support, a wide range of options and support for a bunch of touch devices. It has been tested on Android phones, iPhone & iPad, Windows phone and touch-screen laptops and tablets and desktops; All modern browsers and IE9+ are supported. The end result? A lean, extendible and bloat-less plugin that'll just do its job.
+noUiSlider is lightweight JavaScript range slider, originally developed to be a jQuery UI alternative.
```

Comment:
Took the opportunity to break some lines to make it more readable this cool paragraph